src/js/Controller/Setting/Get.js

```diff
@@ -20,7 +20,9 @@ export default class Get extends AbstractController {
             'server.default',
             'theme.current',
             'theme.custom',
-            'debug.localisation.enabled'
+            'debug.localisation.enabled',
+            'search.recommendation.option',
```

Comment:
It's hard to find a good name for this setting, but "option" just says nothing to me.
Maybe something like "search.recommendation.mode" or "search.recommendation.base" would be more fitting.

Reply:
I agree with you the naming was bad. I changed it to "search.recommendation.mode". But I'm not sure if maybe "search.recommendation.filter" would be even better. What do you think?

Reply:
The reason i suggested "mode" is that we could probably add something like an automatic mode or some other kind of more advanced recommendation later and then this setting name would still make sense.

Reply:
ok sounds logical for me. Thanks for the explanation.